kiwi_post_run

```diff
@@ -98,18 +91,23 @@ echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
 echo "metadata $METADATA"
+echo "tagfile $TAGFILE"
 
 # Generate metada JSON file for container-feeder
 cat << EOF > $METADATA
 {
   "image": {
     "name": "$CONTAINER_RAW",
-    "tags": [ "$CONTAINER_TAG", "latest" ],
+    "tags": [ "${VERSION}-${RELEASE}", "${CONTAINER_TAG}-${RELEASE}", "$CONTAINER_TAG", "latest" ],
     "file": "$IMAGE"
   }
 }
 EOF
 
+# Generate tagfile for automation to determine the latest, most specific, tag for
+# the currently installed RPM
+echo -n "${VERSION}-${RELEASE}" > $TAGFILE
+
```

Comment:
I played with a few iterations of this, and using the VERSION+RELEASE felt best, however, I'm fine with either.. (Though, as with the other PRs, I won't have time to update this.. I'm moments away from going on holidays ;))